docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -11,56 +11,56 @@ tags:
+The **ORDER BY** operator is supported only with [Advanced Visibility](/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster).
 
-- The following operators are supported in List Filters:
+### Partial string match
 
-  - **=, !=, >, >=, <, <=**
-  - **AND, OR, ()**
-  - **BETWEEN ... AND**
-  - **IN**
-  - **ORDER BY**
-  - **LIKE**
+If a `Description` Search Attribute of `Text` type is set to a phrase, List Filters containing words within that phrase will return the Workflow.
+Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
+- `Description="quick"` or `Description="fox"` would both return a Workflow with the Description Search Attribute set to "The quick brown fox jumps over the lazy dog."
+- `Description="qui"` or `Description="laz"` would not return the Workflow.
+  [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
+- To use a partial string match (i.e. specific word), create a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` and set it value to a phrase that contains that word.
 
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+### Efficient API usage
 
-- A List Filter applies to a single Namespace.
+An Advanced List Filter API may take longer to respond if it is retrieving a large number of Workflow Executions (over 10 million, for instance).
 
-- The range of a List Filter timestamp (StartTime, CloseTime, ExecutionTime) cannot exceed 9223372036854775807 (that is, maxInt64 - 1001).
+Use the `CountWorkflow` API to efficiently count the number of [Workflow Executions](/concepts/what-is-a-workflow-execution).
+Paginate the results with the `ListWorkflow` API by using the page token to retrieve the next page; continue until the page token is `null`/`nil`.
 
-- A List Filter that uses a time range has a resolution of 1 ms on Elasticsearch 6 and 1 ns on Elasticsearch 7.
+#### List Filter examples
 
-- List Filter Search Attribute names are case sensitive.
+The following is a List Filter set with [`tctl`](/tctl-v1/workflow/list):
 
-- An Advanced List Filter API may take longer than expected if it is retrieving a large number of Workflow Executions (more than 10 million).
+```
+WorkflowType = "main.YourWorkflowDefinition" and ExecutionStatus != "Running" and (StartTime > "2021-06-07T16:46:34.236-08:00" or CloseTime > "2021-06-07T16:46:34-08:00") order by StartTime desc
```

Comment:
Can we remove the `order by` part? Might cause confusion as in SQL is not supported.